src/bin/pgcopydb/ld_stream.h

```diff
@@ -365,6 +365,12 @@ typedef struct StreamApplyContext
 	/* target connection */
 	PGSQL pgsql;
 
+	/* target connection created in pipeline mode */
+	PGSQL pgsqlPipeline;
+
+	/* track the last time we synced the pipeline */
+	uint64_t pipelineSyncTime;
+
```

Comment:
Again, not fond of the naming. This should be the `PGSQL applyConn`, as opposed to I suppose the `PGSQL controlConn` or something. Also pipelineSyncTime looks like it should belong within the PGSQL structure itself, right?